File: fruitStore/java/src/main/java/cn/bnuz/onlineStore/entity/Goods.java
package cn.bnuz.onlineStore.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author Zhang Hao
 * @create 2022-11-23-15:59
 */
@Data
@TableName("goods_tb")
@NoArgsConstructor
@AllArgsConstructor
public class Goods {
    @TableId(type = IdType.AUTO)
    private Integer goodsId ;
    private String goodsName;
    private Float goodsPrice;
    private Integer goodsStocks;
    private String goodsDescription;
    private String filePath;
}

File: fruitStore/kotlin/app/src/main/java/cn/bnuz/mystore/entity/Goods.kt
package cn.bnuz.mystore.entity

import androidx.versionedparcelable.ParcelField
import java.io.Serializable


data class Goods (
    val goodsId: Int,
    val goodsName: String,
    val goodsPrice: Double,
    val goodsStocks: Int,
    val goodsDescription: String,
    val filePath : String
) :Serializable